python/paddle/v2/framework/nets.py

```diff
@@ -103,22 +103,18 @@ def sequence_conv_pool(input,
                        filter_size,
                        pool_size,
                        pool_stride,
-                       act,
                        program=None,
                        init_program=None):
     conv_out = layers.sequence_conv(
         input=input,
         num_filters=num_filters,
         filter_size=filter_size,
-        act=act,
         program=program,
         init_program=init_program)
 
     pool_out = layers.sequence_pool(
         input=conv_out,
-        pool_size=pool_size,
-        pool_type='max',
-        pool_stride=pool_stride,
+        pool_type='sum',
```

Comment:
I think `max` is a little more normal than `sum`, so I think it's fine to keep the default as `max`.

Reply:
Fixed.
Find a frequent usage -- embedding and then pooling in `Sum` is equivalent to the FC layer.